chainbridge/src/mock.rs

```diff
@@ -109,8 +118,8 @@ frame_support::construct_runtime!(
         UncheckedExtrinsic = UncheckedExtrinsic
     {
         System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-        Balances: pallet_balances::{Pallet, Call, Config<T>, Storage, Event<T>},
-        ChainBridge: pallet_chainbridge::{Pallet, Call, Storage, Event<T>},
+        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+        Chainbridge: pallet_chainbridge::{Pallet, Call, Storage, Event<T>},
```

Comment:
Can we keep it as Camel case? That way we do not have to change any clients that use it

Reply:
Just to be sure. Do clients access mock runtime?

Reply:
It is a reference, and that is how we implemented it on our centrifuge-chain side, just to keep it consistant, and mergeable upstream.

Reply:
Okay. Thanks Miguel for your "lights" :)